code/extensions/Lumberjack.php

```diff
@@ -37,7 +37,8 @@ public function getExcludedSiteTreeClassNames() {
 	public function updateCMSFields(FieldList $fields) {
 		$excluded = $this->owner->getExcludedSiteTreeClassNames();
 		if(!empty($excluded)) {
-			$pages = SiteTree::get()->filter(array(
+			$childClassName = $this->getChildClassName();
+			$pages = $childClassName::get()->filter(array(

```

Comment:
Could you change this to DataList::create($childClassName)->get()...

Just means the framework can control any errors in DataList::create()